local_settings_example.py

```diff
@@ -1,2 +1,6 @@
+# OpenFoodFacts username and password for authentication
 USER='myusername'
 PASSWORD='mypassword'
+# Change Postgres user and host as per requirement
+postgresUser = 'myPostgresUsername'
+postgresHost = 'myPostgresHost' # Usually "localhost"
```

Comment:
Sorry @aadarsh-ram one more change: can you make it:
```python
POSTGRES_USER= 'myPostgresUsername'
POSTGRES_HOST = 'myPostgresHost'
```

This is more python conventions (camel case is used for class not for variable in python, and constants are upper cased).  see https://peps.python.org/pep-0008/#naming-conventions

Reply:
Made it uppercase. Will keep this in mind!